app/models/availability.rb

```diff
@@ -6,18 +6,16 @@
 #
 #  id            :integer          not null, primary key
 #  user_id       :integer
-#  day           :integer          not null
 #  time_range_id :integer
 #  created_at    :datetime         not null
 #  updated_at    :datetime         not null
 #  status        :boolean          not null
+#  day           :integer
 #
 # Indexes
 #
-#  index_availabilities_on_time_range_id                      (time_range_id)
-#  index_availabilities_on_user_id                            (user_id)
-#  index_availabilities_on_user_id_and_time_range_id_and_day
```

Comment:
Did you not have this index in your db?

Reply:
Yeah :/ I realised what I did wrong. Fixed!